dev/lint-python

```diff
@@ -47,11 +49,36 @@ if [ ! -e "$PEP8_SCRIPT_PATH" ]; then
     fi
 fi
 
+# Easy install pylint in /dev/pylint. To easy_install into a directory, the PYTHONPATH should
+# be set to the directory.
+# dev/pylint should be appended to the PATH variable as well.
+# Jenkins by default installs the pylint3 version, so for now this just checks the code quality
+# of python3.
+export "PYTHONPATH=$SPARK_ROOT_DIR/dev/pylint"
```

Comment:
AFAIK the PEP8 script currently suffers from this problem as well.  I'm talking with Shane about configuring some per-build space which lives outside of the work directory that's cleaned by `git clean` (I want to use these for holding separate `.ivy2` caches for each PRB workspace).